static/docroot/index.php

```diff
@@ -104,7 +104,7 @@
 			],
 			[
 				'os'    => 'linux',
-				'url'   => 'https://wiki.wesnoth.org/WesnothBinariesLinux',
+				'url'   => 'https://flathub.org/apps/details/org.wesnoth.Wesnoth',
```

Comment:
What about the development version?

Reply:
As far as I'm aware there isn't a proper web page to link to for getting the dev flatpak - it's just mentioned at the top of the WesnothBinariesLinux page which then links to a forum post by loonycyborg about how to install it.